services/zapier.rb

```diff
@@ -0,0 +1,65 @@
+class Service::Zapier < Service
+  def receive_validate(errors = {})
+    if settings[:url].to_s.empty?
+      errors[:url] = "Is required"
+      return false
+    end
+
+    uri = URI.parse(settings[:url]) rescue nil
+    unless uri
+      errors[:url] = "Is not a valid URL"
+      return false
+    end
+
+    return true
+  end
+
+  def receive_alert
+    raise_config_error unless receive_validate({})
+    send_alert(body)
```

Comment:
@obfuscurity I think it may be ok to not support v1 alerts.  They are in most ways officially deprecated -- they can't be created, and only existing ones may be modified.  The slack librato-service [does not support](https://github.com/librato/librato-services/blob/master/services/slack.rb#L139) v1 alerts for example.

The correllary to not supporting v1 alerts for zapier is to ensure that the zapier service does not appear in the v1 alerts service selector (available from the legacy metric page in the alert tab) and also to ensure that the api enforces this as well (i.e. rejects alerts from being persisted which are v1 alerts but use slack/zapier/etc services).